File: co.edu.uniandes.fixml.toCSharp/test/test3.xml/LimitOrder.cs

using System.Collections.Generic;
// Class for tag LimitOrder
class LimitOrder {

	string _Value;
	
	List<Price> _Prices = new List<Price>();
	
	// default constructor
	
	public LimitOrder ( ) {   
		this._Value = "2";
		
		Price LimitOrder_Price= new Price();
		this.AddPrice( LimitOrder_Price );
		
		
	}
	
	public LimitOrder ( string Value ) {
		this._Value = Value;
		
	}
	
	public void AddPrice( Price child ) {
		this._Prices.Add( child );
	}
	
	
} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/Java/LimitOrder.java
package object;

import java.util.List;
import java.util.ArrayList;

// Class for tag LimitOrder
public class LimitOrder {


  List<Price> Prices = new ArrayList<Price>();



  public LimitOrder ( ) {   
  
;
  
Price LimitOrder_Price_0= new Price("0", "99999999.9999");
this.addPrice( LimitOrder_Price_0 );
  		
  }
      
  public void addPrice( Price child ) {
    this.Prices.add( child );
  }

} 